README.md

```diff
@@ -1,44 +1,88 @@
-Guides
-======
```

Comment:
I like that you're converting over to GitHub-flavored Markdown. I've had updating our Guides to match this as a thing I wanted to do for awhile now.

Reply:
I didn't want to do it for all the files as part of this PR, but I want to do a follow-up "formatting only" PR to address this for all files 🙂

Reply:
Just created #617!